File: SDKs/vimeo.com/3.4/python/src/sdk/models/shared/domain.py
from dataclasses import dataclass, field
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclass
class Domain:
    allow_hd: bool = field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('allow_hd') }})
    domain: str = field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('domain') }})
    uri: str = field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('uri') }})
    

File: SDKs/vimeo.com/3.4/typescript/src/sdk/models/shared/domain.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class Domain extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=allow_hd" })
  allowHd: boolean;

  @SpeakeasyMetadata({ data: "json, name=domain" })
  domain: string;

  @SpeakeasyMetadata({ data: "json, name=uri" })
  uri: string;
}
